server/client.go

```diff
@@ -220,6 +221,7 @@ type client struct {
 	msgb    [msgScratchSize]byte
 	last    time.Time
 	parseState
+	headers bool
```

Comment:
We make it a dedicated bool because we are going to access this without lock in the parser/readloop, right?

Reply:
Yes, will make more clear.